students/JaNiece/index.css

```diff
@@ -0,0 +1,69 @@
+body{
```

Comment:
🙃 Typically we add a space in between the CSS selector and the opening curly brace